content/docs/welcome-guide/create/creating-projects-using-project-manager.md

```diff
@@ -18,11 +18,11 @@ At the end of the tutorial, you'll have a new O3DE project based on the default
 
```

Comment:
Minor fix for line 11: Change `**O3DE Project Manager**` to `**Project Manager**`.